planckton/init.py

```diff
@@ -1,25 +1,78 @@
 import numpy as np
-
 import mbuild as mb
 import parmed as pmd
-from planckton.utils import base_units
+import unyt as u
+from unyt.exceptions import UnitConversionError
+
+from planckton.utils.base_units import planckton_units
 
 
 class Compound(mb.Compound):
-    """ Wrapper class for mb.Compound"""
+    """
+    Wrapper class for mb.Compound
+
+    Parameters
+    ----------
+    path_to_mol2 : str
+        Path to the mol2 file to load
+
+    Attributes
+    ----------
+    mass : unyt.unyt_quantity
+        The mass of the compound in amus
+    """
 
     def __init__(self, path_to_mol2):
         super(Compound, self).__init__()
         mb.load(path_to_mol2, compound=self)
         # Calculate mass of compound
-        self.mass = np.sum([atom.mass for atom in self.to_parmed().atoms])
+        # ParmEd uses amu
+        # TODO use ele?
+        self.mass = np.sum([a.mass for a in self.to_parmed().atoms]) * u.amu
         # We need to rename the atom types
         compound_pmd = pmd.load_file(path_to_mol2)
         for atom_pmd, atom_mb in zip(compound_pmd, self):
             atom_mb.name = "_{}".format(atom_pmd.type)
 
 
 class Pack:
+    """
+    Convenience class for filling box and atomtyping
+
+    Parameters
+    ----------
+    compound : Compound or list of Compounds
+        Compound(s) to initialize in simulation
+    n_compounds : int or list of ints
+        Number(s) of compound(s) to initialize
+    density : float or unyt.unyt_quantity
```

Comment:
I think this should be only a `unyt.unyt_quantity` and error on float

Reply:
you got it 👍